force-app/main/default/chat/lwc/chat/chat.css

```diff
@@ -0,0 +1,3 @@
+.history{
+    margin-left: 20px;
```

Comment:
Why didn't you add a margin to the right side too?

Reply:
added margin to the right 8d870610e229cc8a420aa5daeddcdb7010ca08a5